core/block.go

```diff
@@ -111,15 +111,21 @@ func (mw *Middleware) ObjectShow(cctx context.Context, req *pb.RpcObjectShowRequ
 		return m
 	}
 
+	id := domain.FullID{
+		SpaceID:  req.SpaceId,
+		ObjectID: req.ObjectId,
+	}
 	err := mw.doBlockService(func(bs *block.Service) (err error) {
-		id := domain.FullID{
-			SpaceID:  req.SpaceId,
-			ObjectID: req.ObjectId,
-		}
 		obj, err = bs.ShowBlock(id, req.IncludeRelationsAsDependentObjects)
 		return err
 	})
-
+	if obj != nil {
+		blocksModifiers, err := getService[history.History](mw).GetBlocksModifiers(id, "", obj.GetBlocks())
```

Comment:
This is not very good, because it is too cumbersome. Let's discuss on sync, but basically we need to do this on regular tree build + save authors in snapshots.